pkg/registry/batch/job/strategy_test.go

```diff
@@ -730,6 +730,38 @@ func TestJobStrategy_ValidateUpdate(t *testing.T) {
 				job.Annotations["hello"] = "world"
 			},
 		},
+		"old job with 0 in failure policy (operator In)": {
+			job: &batch.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:            "myjob",
+					UID:             "test",
+					Namespace:       metav1.NamespaceDefault,
+					ResourceVersion: "10",
+					Annotations:     map[string]string{"foo": "bar"},
```

Comment:
Drop fields which are not required for the test, I'm pretty sure "Annotations" and "UID" are not needed.